docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -440,18 +406,20 @@ Logging Driver: json-file
 [...]
 ```
 
-The `Data Space` values show that the pool is 100GiB total. This example extends the pool to 200GiB.
+The `Data Space` values show that the pool is 100GB total. This example extends the pool to 200GB.
 
 1. List the sizes of the devices.
 
 	```bash
-	$ sudo ls -lh /var/lib/docker/devicemapper/devicemapper/
-	total 1.2G
-	-rw------- 1 root root 100G Apr 14 08:47 data
-	-rw------- 1 root root 2.0G Apr 19 13:27 metadata
+	$ sudo ls -al /var/lib/docker/devicemapper/devicemapper/
+	total 1175492
+	drwx------ 2 root root         4096 Mar 29 02:45 .
+	drwx------ 5 root root         4096 Mar 29 02:48 ..
+	-rw------- 1 root root 107374182400 Mar 30 05:22 data
+	-rw------- 1 root root   2147483648 Mar 31 11:17 metadata
```

Comment:
Did you have a reason to use raw numbers here, and not the "human friendly" ones? (`ls -lh`)